content/handbook/sales/training/sales-enablement-sessions/_index.md

```
@@ -0,0 +1,515 @@
+
+Field Enablement Spotlight セッションは、Sales and Customer Enablement と PMM チームの全員がカレンダーイベントを編集できるよう、[Sales Enablement カレンダー](https://calendar.google.com/calendar/embed?src=gitlab.com_5n3g60l58thum9aovp8iisav34%40group.calendar.google.com&ctz=America%2FLos_Angeles)でスケジュール設定されています。
+
+Field Enablement Spotlight は毎週木曜日に開催され、グローバルな GitLab フィールドセールスのオーディエンスに対応するため、2つのセッションオプションを用意しています。
+
+- **(オプション1) 11:00 AM (米国東部時間)(UTC -5)** = (午前8時 太平洋時間) = **(午後5時 中央ヨーロッパ時間)**
+
+- **(オプション2) 12:00 PM (オーストラリア東部時間)(UTC +11)** = (午後5時 米国太平洋時間)
+
+## Field Enablement Spotlight セッションの実施方法
+
+- 各トレーニングセッションには [スピーカー](#speaker) と [モデレーター](#moderator) がいます
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)